includes/admin/class-wp-job-manager-admin-notices.php

```diff
@@ -350,6 +389,37 @@ private static function render_notice( $notice ) {
 			echo '</div>';
 		}
 		echo '</div>';
+
+	}
+
+	/**
+	 * Get the dismissed notifications (either for the user or site-wide).
+	 *
+	 * @param bool $is_user_notification True if this is for a user notification (vs site-wide notification).
+	 *
+	 * @return array
+	 */
+	private function get_dismissed_notices( $is_user_notification ) {
+		// TODO Implement from Sensei.
+	}
+
+	/**
+	 * Save dismissed notices.
+	 *
+	 * @param array $dismissed_notices Array of dismissed notices.
+	 * @param bool  $is_user_notification True if we are setting user notifications (vs site-wide notifications).
+	 */
+	private function save_dismissed_notices( $dismissed_notices, $is_user_notification ) {
+		// TODO Implement from Sensei.
+	}
```

Comment:
Can you remove these? Since we are not using them we don't really need these skeletons here.

Reply:
Committed https://github.com/Automattic/WP-Job-Manager/pull/2401/commits/f82515acd805ecddeac7434e5f7a66de48f4c4ca